docs/known-issues.md

```diff
@@ -29,6 +29,20 @@ This seems to be caused by the Keychain being in an invalid state, affecting app
 - Right-click on the `login` keychain and try unlocking it
 - Sign into your GitHub account again
 
+### Checking for updates triggers a 'Could not create temporary directory: Permission denied' message = [#4115](https://github.com/desktop/desktop/issues/4115)
```

Comment:
Did you mean to put a `-` here instead of an `=`?

Reply:
Sure did!